### PR TITLE
Use Makefile.COMMON

### PR DESCRIPTION
**Pull request by discordianfish**
This now uses the shared Makefile.COMMON but since the alertmanager requires to run make -C web before building, it doesn't work. We'll have the same problem with prometheus itself.

So how should we solve that? We could make it possible to execute arbitrary build steps by changing the Makefile.COMMON, we could commit the files.go or we use go-generate.

Related: https://github.com/prometheus/prometheus/issues/548

### CODE REVIEW

**Comment by discordianfish** [comment]
So for now I added the files.go, I guess we should discuss in https://github.com/prometheus/prometheus/issues/548 what to do on the long run.

File: web/Makefile [hunk 1]
```diff
@@ -11,16 +11,14 @@
...
 
-include ../Makefile.INCLUDE
-
 all: blob/files.go
 
 SUFFIXES:
 
 blob/files.go: $(shell find templates/ static/ -type f)
-	./blob/embed-static.sh static templates | $(GOFMT) > $@
+	./blob/embed-static.sh static templates | gofmt > $@
```
Comment (juliusv) [inline web/Makefile]: If we kept the `make web` step in the parent Makefile (to be executed manually), we could leave this as is, no?

File: Makefile [hunk 2]
```diff
@@ -11,47 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include Makefile.INCLUDE
+VERSION  := 0.1.0
```
Comment (juliusv) [inline Makefile]: With alertmanager as an exception, I would go for 0.0.1 to point out to people just how experimental it is...

**Comment by brian-brazil** [comment]
:+1:

**Comment by juliusv** [comment]
I'd include the Makefile.COMMON at the top, but otherwise :+1:

**Pull request merged**